include/DeviceProvider.h

```diff
@@ -14,6 +14,7 @@
 
 #include "Encode/EncodingManager.h"
 #include "Encode/LegacyEncodingManager.h"
+#include "Encode/AlphaEncodingManager.h"
 
 #include "DeviceDriver/DeviceDriver.h"
```

Comment:
I know this doesn't really relate to the PR, but can we move these includes into `DeviceProvider.cpp`? There's not really a reason for them to be in the header, as they aren't used here.

Reply:
Sure thing